lib/matplotlib/cbook.py

```diff
@@ -2327,3 +2327,95 @@ def get_instancemethod(self):
 else:
     def _putmask(a, mask, values):
         return np.copyto(a, values, where=mask)
+
+class OrderedDict(dict):
```

Comment:
Ahh, I didn't know that that existed, I shall copy-paste.  Do you know what the dev's think?